whitelabelco/sofo-foods
# Cp/feat/product images update

### Description 💻

Resolves: missing asset for "Dairy"

### Implementation 💡

Updates asset for Dairy and adds thumbails for product images

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Products"
- confirm Dairy asset appears in sidebar and image grid
- click "Dairy"
- confirm Dairy hero image appears

### Screenshots/Demos 📷

### Checklist ✅
